src/web/app/components/question-responses/gqr-rqg-view-responses/gqr-rqg-view-responses.component.ts

```diff
@@ -83,6 +85,10 @@ export class GqrRqgViewResponsesComponent extends ResponsesInstructorCommentsBas
             this.teamsToUsers[response.giverTeam].push(response.giver);
             this.teamExpanded[response.giverTeam] = this.isExpandAll;
           }
+          this.userToEmail[response.giver] = this.userToEmail[response.giver] || '';
```

Comment:
This is no necessary if you have `*ngIf="userToEmail[userInfo]"`?

Reply:
done